### DISCUSSION TITLE
android emulator

### DISCUSSION THREAD

**Discussion by sajjad000** [linuxdeepin/developer-center#3516] [Q&A | 问答板块]
How hide the side menu bar under the screen in deepin does not allow but Windows allows to hide the bar under the screen 
![BAjo5](https://user-images.githubusercontent.com/118594121/206837551-1cbc1bec-fea0-43fe-84b4-d8bb6b44e55c.png)

**Comment by BLumia** [comment]
Clicking the close "x" (close) button on that sidebar doesn't work as intended? Could you give me more details so we could try to reproduce the issue? e.g. Android Studio version, deepin version, etc.

Thanks!

**Comment by sajjad000** [comment]
Do you have an account to send you a video

**Reply by BLumia** [reply]
You can directly post the video to here (or somewhere like YouTube and link it here) :)

**Reply by sajjad000** [reply]
In Windows I can hide the menu bar but deepin doesn't allow it

https://user-images.githubusercontent.com/118594121/206851959-1528b7ee-9d11-4173-9a06-d561524e3600.mp4

**Reply by BLumia** [reply]
Could you attach a video to reproduce the issue in deepin? The information I asked previously also include: Android Studio / Android Emulator version, and deepin version.

Thanks!

**Reply by sajjad000** [reply]
I installed deepin 20.8 and android Studio 2021.3.1 look at the problem deepin refuses to enter the menu under the screen while windows allows it



https://user-images.githubusercontent.com/118594121/206885219-b6e95104-b526-432e-83f2-a67b0c9f05b3.mp4
